### PR TITLE
Implement automatic expiration of sessions.

### PR DESCRIPTION
**Pull request by dantman**

### CODE REVIEW

File: lib/connect-session-sequelize.js [hunk 1]
```diff
@@ -25,7 +29,20 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
+		var key;
+		this.options = {};
+		for ( key in defaultOptions ) {
+			this.options[key] = defaultOptions[key];
+		}
+		for ( key in options ) {
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: could you remove the whitespace in the brackets and before the bracket?

Comment (dantman) [inline lib/connect-session-sequelize.js]: Oh right, I forgot the convention in half the commit.

File: lib/connect-session-sequelize.js [hunk 2]
```diff
@@ -25,7 +29,20 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
+		var key;
+		this.options = {};
+		for (key in defaultOptions) {
```
Comment (mweibel) [inline lib/connect-session-sequelize.js]: I don't really understand what you're doing  with these `defaultOptions`? Couldn't you use a `extend`, e.g.:
```javascript
util._extend({foo: 'bar'},{bar:'baz',foo:'x'})
```

**Comment by mweibel** [comment]
Thanks, looks good otherwise :)
Would you mind editing the README to add a hint that there's session expiry built-in and how to extend the interval (or how to disable the expiry)?

File: lib/connect-session-sequelize.js (hunk 2 above)
Comment (dantman) [inline lib/connect-session-sequelize.js]: I wasn't aware of util._extend and didn't want to add a library you didn't already have.

Comment (mweibel) [inline lib/connect-session-sequelize.js]: Ah ok. Yeah as far as I know it's undocumented (don't really know why) but works perfectly :)

**Comment by mweibel** [comment]
ping?

**Comment by dantman** [comment]
Sorry, I've been working on other stuff. And some recent changes mean that I'll likely be switching to conect-dynamodb for sessions instead of changing what mysql session store I use.

**Comment by jakeorr** [comment]
This has been brought up to date: https://github.com/mweibel/connect-session-sequelize/pull/19

**Pull request merged**